recipes/libnvjpeg/meta.yaml

```
@@ -0,0 +1,148 @@
+{% set name = "libnvjpeg" %}
+{% set version = "12.0.0.28" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: ad3380591a4d5b97ab57d10e9bf05e23dee6429ff91ea939423a3b35ee0e35de  # [linux64]
+  sha256: f7136bb691d794c680d77376b306bedb80eeebf687bfa4dc74355fdae5adcebb  # [ppc64le]
+  sha256: 31fd6ee0dccc72f0d583e25b849543551eb4bc49d3285628bf3b17929e09bac1  # [aarch64]
+  sha256: b7cb56ab2c69e055a4b1ea41653df917153a568ab23fb0d8121e0739011989a8  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
```

We will have to skip Windows for now because the dependency `cuda-cudart-dev` is not yet available on Windows.
```suggestion
  skip: true  # [osx or win]
```